config/jobs/image-pushing/k8s-staging-nfd.yaml

```diff
@@ -0,0 +1,31 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
+  kubernetes-sigs/node-feature-discovery:
+    # The name should be changed to match the repo name above
+    - name: post-node-feature-discovery-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        # This is the name of some testgrid dashboard to report to.
+        # If this is the first one for your sig, you may need to create one
+        testgrid-dashboards: sig-node-node-feature-discovery
```

Comment:
This PR is failing because this dashboard isn't in https://github.com/kubernetes/test-infra/blob/master/config/testgrids/kubernetes/sig-node/config.yaml

Reply:
Thanks for the review. I now added dashboard in the testgrid config.